guides/micronaut-scope-types/kotlin/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+#Mon Sep 18 18:49:56 CEST 2023
```

Comment:
delete this file. This file will be automatically generated.

Reply:
removed